README.md

```diff
@@ -11,7 +11,17 @@ It contains [Lightning](https://github.com/rdkcentral/Lightning) and several use
 - Various helpers
 
 Check out the complete [SDK documentation](https://rdkcentral.github.io/Lightning-SDK/) for more information.
+## Installation
 
+Install with NPM
+```bash
+npm install -g @lightningjs/cli
+```
+
+Install with Yarn
+```bash
+yarn global add @lightningjs/cli
+```
```

Comment:
These are instructions to install Lightning-CLI. I don't see the reasoning to include this on the Lightning-SDK readme file.

Reply:
I agree with this as well. I know we can always do better with documentation and installation instructions, but I don't think these exact instructions are the best addition.

Instead we could consider adding a snippet referring the Getting Started section of the docs on LightningJS.io: https://lightningjs.io/docs/#/getting-started/index